status.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+buildifier_tags=$(git describe --tags)
+IFS='-' read -a parse_tags <<< "$buildifier_tags"
+if [[ $? != 0 ]];
```

Comment:
`set -e` should cause the script to exit with a failure status when any later command exits with a failure status. That frees you from having to check `$?`.